### PR TITLE
I175 contacts page update

### PR DESCRIPTION
**Pull request by QuantumApostle07**
Changed delete contact to logout in the user profile (for signed-in users)

## Change Summary
**[Briefly explain the changes that you made. Just high-level stuff]**

Duplicated the 'delete contact' button and modified it to 'Logout'
Used async and logout function

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**
No

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,4 +51,10 @@ sitemap-*.xml
...
 **/public/workbox-*.js.map
-**/public/worker-*.js.map
\ No newline at end of file
+**/public/worker-*.js.map
+*.zip
+*.yml
+*.gz
+*.lock
+*.lock
+yarn.lock
```
Comment (scratchclaggy) [inline .gitignore:60]: Why did you add all of these to the `gitignore`? We want the `yarn.lock` file committed (https://stackoverflow.com/a/39992365).

File: src/components/Contact/contactinfo.tsx [hunk 2]
```diff
@@ -147,12 +152,27 @@ function ContactInfo() {
...
             Delete Contact
           </Button>
-        )}
+        ) : (
+          <Button
+            type="button"
+            onClick={async () => {
+              try {
+                const result = await signOut(auth) //TODO
+                console.log(result);
```
Comment (scratchclaggy) [inline src/components/Contact/contactinfo.tsx:159]: Remove console log statements

File: src/context/Firebase/Auth/index.tsx [hunk 3]
```diff
@@ -69,14 +69,13 @@ export const AuthContextProvider = ({ children }: { children: ReactNode }) => {
...
-      .then((result) => {
-        return result
-      })
-      .catch((error) => {
-        return error
-      })
+  async function logOut() {
+    try {
+      const result = await signOut(auth)
+      return result
```
Comment (scratchclaggy) [inline src/context/Firebase/Auth/index.tsx:75]: You don't need `result` here, i.e.:
```typescript
      return await signOut(auth)
```

**Review by scratchclaggy — CHANGES_REQUESTED** [review]
Just a couple little things here, plus some merge conflicts to resolve. Also, I figured out why the `docId !== 'USER'` wasn't working so I did a little fix for you.

File: .gitignore (hunk 1 above)
Comment (skyheat) [reply .gitignore:60]: Possibly my bad, I must've not noticed that yarn.lock was one of the files to be committed in the vs code version control panel. I offhandedly said to only commit the file he had changed. Oopsies

**Review by scratchclaggy — APPROVED** [review]
Looks good :rocket:
